.ado/windows-vs-pr.yml

```diff
@@ -455,7 +487,8 @@ jobs:
           platform: $(BuildPlatform)
           configuration: $(BuildConfiguration)
           publishRunAttachments: true
-          collectDumpOn: onAbortOnly
+          collectDumpOn: always
```

Comment:
>always [](start = 25, length = 6)

I wonder what it means to collect dumps always. Does it collect on successful runs too?

Reply:
From the [docs](https://docs.microsoft.com/en-us/azure/devops/pipelines/tasks/test/vstest?view=azure-devops)


collectDumpOnCollect process dump and attach to test run report | (Optional) Use this option to collect a mini-dump that can be used for further analysis.On abort only: mini-dump will be collected only when test run is aborted.Always: mini-dump will always be collected regardless of whether the test run completes or not.Never: mini-dump will not be collected regardless of whether the test run completes or not
-- | --